host/windows/syscall.c

```diff
@@ -723,5 +1144,25 @@ int oe_syscall_getgroups_ocall(size_t size, unsigned int* list)
 
 int oe_syscall_uname_ocall(struct oe_utsname* buf)
 {
-    PANIC;
+    // Based on
+    // https://docs.microsoft.com/en-us/windows/win32/sysinfo/getting-the-system-version
+    // OE SDK is supported only on WindowsServer and Win10
+    if (IsWindowsServer())
```

Comment:
Is there a better implementation of uname?

Reply:
Was there a reason you populated the information the way you did? For instance, if I populated it like this:

sysname: Windows
version: 10 (or Server2016, Server2019, etc.)
release: <windows build number>

Where is this information going to be used?